src/admin-facility-list/messages_en.json

```diff
@@ -0,0 +1,15 @@
+{
+    "adminFacilityList.facilities": "Facilities",
+    "adminFacilityList.geographicZone": "Geographic zone",
+    "adminFacilityList.noFacilities": "No facilities found.",
+    "adminFacilityList.name": "Facility name",
+    "adminFacilityList.code": "Code",
+    "adminFacilityList.type": "Type",
+    "adminFacilityList.active": "Active",
+    "adminFacilityList.enabled": "Enabled",
+    "adminFacilityList.actions": "Actions",
+    "adminFacilityList.search": "Search",
+    "adminFacilityList.addFacility": "Add Facility",
+    "adminFacilityList.edit": "Edit",
+    "adminFacilityList.print": "Print"
+}
```

Comment:
There is no need to copy the whole messages_en.json file, you can just add  "adminFacilityList.print": "Print"

Reply:
ok, I will change this